src/components/SideBarLeft.tsx

```diff
@@ -5,10 +5,11 @@ import StepList, { StepListProps } from './StepList';
 
 /* tslint:disable-next-line */
 const SideBarRightWrapper = styled.div`
-  width: 240px;
+  width: 220px;
   background-color: rgba(216, 216, 216, 0.1);
   height: 637px;
   margin-top: 90px;
```

Comment:
I think this `margin-top` is too big, making the sidebar a little bit low.

Reply:
Ok, I'll try another series of `margin-top`.